bin/adhoc-sign.py

```diff
@@ -0,0 +1,67 @@
+#! /usr/bin/env python
+
+import argparse
+import hashlib
+
+import yaml
+import requests
+
+
+def main(args: argparse.Namespace):
+    release = args.release[0]
+    filename = "mozregression-gui-app-bundle.tar.gz"
+    urls = {
+        "macOS": "https://github.com/mozilla/mozregression"
+        f"/releases/download/{release}/{filename}",
+        "windows": "https://github.com/mozilla/mozregression"
+        f"/releases/download/{release}/mozregression-gui.exe",
+    }
+
+    # Windows signing is not yet supported by adhoc-signing for mozregression.
+    signing_formats = {
+        # "windows": ["autograph_authenticode"],
+        "macOS": ["macapp"],
+    }
+
+    params = {}
+
+    for signing_format in signing_formats:
```

Comment:
Use `for signing_format, app in signing_formats.items()` to avoid `signing_formats[signing_format]` below.

Reply:
Renamed the variables to make more sense.